scipy/stats/tests/test_mstats_basic.py

```diff
@@ -420,6 +420,17 @@ def test_mode(self):
         assert_equal(mstats.mode(ma4, axis=0), ([[3,2]], [[1,1]]))
         assert_equal(mstats.mode(ma4, axis=-1), ([[2],[3],[5]], [[1],[1],[1]]))
 
+        a1_res = mstats.mode(a1, axis=None)
+        assert_equal(str(a1_res)[:4], 'Mode')
+
+        # test namedtuple attributes by index
+        attributes = ('mode', 'count')
+        for i, attr in enumerate(attributes):
+            assert_equal(a1_res[i], getattr(a1_res, attr))
+
+        # test namedtuple attributes explicitly by name
+        assert_equal(a1_res.mode, getattr(a1_res, 'mode'))
+        assert_equal(a1_res.count, getattr(a1_res, 'count'))
```

Comment:
It's not clear to me what this is testing? Does it do anything beyond what the 3 lines above it do?

Rationale- if `getattr(a1_res, 'mode')` doesn't raise `AttributeError` then it should always be true that `a1_res.mode == getattr(a1_res, 'mode'))`

Also, this could be in it's own test function.